moveos/moveos-types/src/moveos_std/table.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    addresses::MOVEOS_STD_ADDRESS,
+    state::{MoveStructState, MoveStructType},
+};
+use move_core_types::{
+    account_address::AccountAddress, ident_str, identifier::IdentStr, language_storage::StructTag,
+    value::MoveStructLayout,
+};
+use serde::{Deserialize, Serialize};
+
+pub const MODULE_NAME: &IdentStr = ident_str!("table");
+
+#[derive(Eq, PartialEq, Debug, Clone, Deserialize, Serialize)]
+pub struct TablePlaceholder {}
```

Comment:
`TypeTable.move` also needs to define `TablePlaceholder` in Rust? Will it be repeated?

Reply:
Yes, but it is not used in Rust now.